src/DIRAC/WorkloadManagementSystem/scripts/dirac_wms_get_wn_parameters.py

```diff
@@ -6,8 +6,6 @@
 from __future__ import division
 from __future__ import print_function
```

Comment:
Is it still necessary here?

Reply:
I have not removed the py2 compatibility from some modules (like this one) that could potentially be invoked in a python2 environment. These include some scripts that are invoked by the `Pilot`, like this one. I really doubt that it will ever be necessary, but there might be some corner cases where this would still be necessary (e.g. running the Pilot "by hand" on certain HPC's WNs). It might well be that I am overthinking this case, but for the moment I left it open. 
Anyway, this won't be the last PR with "py3 fixes/py2 removal".